apps/internal/base/internal/storage/items.go

```diff
@@ -72,6 +72,7 @@ type AccessToken struct {
 	ClientID          string            `json:"client_id,omitempty"`
 	Secret            string            `json:"secret,omitempty"`
 	Scopes            string            `json:"target,omitempty"`
+	RefreshIn         internalTime.Unix `json:"refresh_in,omitempty"`
```

Comment:
We need to keep the cache consistent between MSALs and the property for refresh in `RefreshOn` (similar to ExpiresOn)